app.js

```diff
@@ -138,17 +138,22 @@ if (Config.crashguard) {
 		if (crashMessage !== 'lockdown') return;
 		let stack = Chat.escapeHTML(err.stack).split("\n").slice(0, 2).join("<br />");
 		if (!Rooms.global.lockdown) {
-			if (Rooms.lobby) {
-				Rooms.lobby.addRaw('<div class="broadcast-red"><b>THE SERVER HAS CRASHED:</b> ' + stack + '<br />Please restart the server.</div>');
-				Rooms.lobby.addRaw('<div class="broadcast-red">You will not be able to start new battles until the server restarts.</div>');
-				Rooms.lobby.update();
-			}
 			let staffRoom = Rooms('staff');
 			if (staffRoom) {
 				staffRoom.addRaw('<div class="broadcast-red"><b>THE SERVER HAS CRASHED:</b> ' + stack + '<br />Please restart the server.</div>');
 				staffRoom.addRaw('<div class="broadcast-red">You will not be able to start new battles until the server restarts.</div>');
 				staffRoom.update();
 			}
+			let devRoom = Rooms('development');
+			if (devRoom) {
+				devRoom.addRaw('<div class="broadcast-red"><b>THE SERVER HAS CRASHED:</b> ' + stack + '<br />Please restart the server.</div>');
+				devRoom.addRaw('<div class="broadcast-red">You will not be able to start new battles until the server restarts.</div>');
+				devRoom.update();
+			} else if (Rooms.lobby) {
```

Comment:
If a crash requires a lockdown, the lockdown needs to be displayed everywhere.

If you don't have the crash message itself, at least display a generic lockdown message. I concede that Lobby doesn't need a special case, though, because every room needs it.